facebook_connector.py

```diff
@@ -0,0 +1,41 @@
+import os
+import fnmatch
+import pandas as pd
+from fbchat_archive_parser.parser import MessageHtmlParser
+
+def initialize(location = '.'):
```

Comment:
The root of the facebook dump actually includes index.htm, let's request that location, and perhaps name the variable dump_location